worker/browser-routes.ts

```diff
@@ -1,5 +1,5 @@
 import { Router, Request, Response } from "express";
-import { BrowserManager } from "../services/BrowserManager";
+import { BrowserManager } from "./browser-manager";
 
 export function browserRoutes(browserManager: BrowserManager): Router {
```

Comment:
Delete BrowserManager and attach the browser routes directly to the AgentServer. This means we could clean up BROWSER_PORT to so that we only have one web server going